Source/Core/Core/CoreTiming.cpp

```diff
@@ -385,6 +394,21 @@ void CoreTimingManager::SleepUntil(TimePoint time_point)
 
 void CoreTimingManager::Throttle(const s64 target_cycle)
 {
+  const TimePoint time = Clock::now();
+
+  const bool already_throttled =
+      m_throttled_after_presentation.exchange(true, std::memory_order_relaxed);
+
+  // When Immediate XFB is enabled, try to Throttle just once after each presentation.
+  //  This lowers latency by speeding through to the next presentation after grabbing input.
+  // Make sure we don't get too far ahead of proper timing though,
+  //  otherwise the emulator unreasonably speeds through loading screens that don't have XFB copies.
+  const bool skip_throttle = already_throttled && g_ActiveConfig.bImmediateXFB &&
+                             ((GetTargetHostTime(target_cycle) - time) < (m_max_fallback / 2));
```

Comment:
Out of curiosity, is there a reason we used half of our fallback here?

Reply:
The logic below will push the throttle forward if `m_max_fallback` is exceeded, and I didn't really want to ride the line of that potentially happening. But thinking about this more, this might make more sense just being a fixed value of 50ms (which so happens to be the default value of `m_max_fallback / 2`). That would allow this speeding through to work down do 20fps.

Reply:
I'll also mention that the previously mentioned below logic that pushes the throttle forward if the emulator runs too fast is maybe weird and should just be removed. @Sam-Belliveau  ?